examples/dsa_hisa/block_sparse_mqa_fp8.py

```
@@ -0,0 +1,335 @@
+
+def ref_fp8_block_sparse_mqa(
+    q_fp8: torch.Tensor,
+    k_fp8: torch.Tensor,
+    k_scale: torch.Tensor,
+    topk_block_index: torch.Tensor,
+    kv_block_size: int,
+    weights: torch.Tensor,
+    cu_seqlen_ks: torch.Tensor,
+    cu_seqlen_ke: torch.Tensor,
+) -> torch.Tensor:
+    M, H, D = q_fp8.shape
```

Ruff (0.15.10)

[warning] Unpacked variable `H` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)